integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -763,6 +773,8 @@ void testTwoDomainsInSameNameSpaceOnOperator() {
       HelmParams opHelmParams = installAndVerifyOperator(op2Namespace, opServiceAccount, true,
           0, op1HelmParams, domain4Namespace).getHelmParams();
       assertNotNull(opHelmParams, "Can't install operator");
+      String opExtRestRouteHost = createRouteForOKD("external-weblogic-operator-svc", op2Namespace);
```

Comment:
I don't see opExtRestRouteHost is being referenced in this class. Can you explain the usage of it.

Reply:
OK, Bhavani just explained to me on Slack.